website/www/site/content/en/documentation/io/built-in/webapis.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "Web Apis I/O connector"
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+[Built-in I/O Transforms](/documentation/io/built-in/)
+
+# Web APIs I/O connector
+
+{{< language-switcher java py go >}}
+
+The Beam SDKs include a built-in transform, called RequestResponseIO that can read from and write to Web APIs such as
+REST or gRPC.
```

Comment:
I think it would be helpful to call out some of the features this includes here (throttling, caching, etc...)

Reply:
@damccorm I want such features in future PRs to make it easier to review. This PR should just have the basics.

Reply:
I'm just suggesting adding a sentence here, not building out full examples. I think this sentence as written doesn't convey the full value of the feature.